Update Roadmap-and-Backlog.md (#758)
raft-tech/TANF-app: docs/Product-Strategy/Roadmap-and-Backlog.md

# Product Roadmap

> Our [roadmap](https://app.mural.co/t/officeoffamilyassistance2744/m/gsa6/1592336604317/61159efd030645a74c267130ea19b2083f87dd09) represents our latest thinking about the order in which we’ll tackle the various pieces of the overarching problem.

Reviewer: ```suggestion
Our [roadmap](https://app.mural.co/t/officeoffamilyassistance2744/m/gsa6/1592336604317/61159efd030645a74c267130ea19b2083f87dd09) 🔒  represents our latest thinking about the order in which we’ll tackle the various pieces of the overarching problem.
```

**_This roadmap is not a promise. The order of these items is likely to change as we discuss options and learn more._**

## ATO & OFA MVP Scope
Our first release will produce an application that could be used by OFA Admin staff, but our primary goal is to get Authority to Operate (ATO), which will allow us to have a production environment. Below is a list of functionality that will be included in this release.

| Functionality | Status | 
| -------- | ------- | 
| User can log in using login.gov | Complete
| Users with appropriate privileges can manage users | Complete
| Users can upload data files by section and quarter | In Progress
| Users with appropriate privileges can download files that were previously uploaded     | In Progress

> ## Tribe MVP Scope

Reviewer: ```suggestion
## Tribal MVP Scope
```

> The next release will focus on enabling a subset of tribes to use our the upload functionality. This will allow our application to privide value to one of our key user groups early on. We are currently in the research portion of understanding what this could produce. 

Reviewer: ```suggestion
The next release will focus on enabling a subset of tribes to use the upload functionality. This will allow our application to provide value to one of our key user groups early on. We are currently in the research portion of understanding what this could look like. 
```

> ## Potential future work

Reviewer: ```suggestion
## STT MVP scope
```

Reviewer: I'd like to include some of the user stories/epics that we know we'll be building.
